### PR TITLE
(maint) update version, fix fetch depth

### PR DESCRIPTION
**Pull request by joshcooper**

### CODE REVIEW

**Comment by joshcooper** [comment]
@LivingInSyn The snyk vanagon action [failed with](https://github.com/puppetlabs/puppet-runtime/runs/5211711881?check_suite_focus=true#step:4:1227)

```
Error: No space left on device : '/home/runner/runners/2.287.1/_diag/pages/67316a03-e358-4d3e-ac40-4186455a786e_04cbd90d-f885-551f-8636-bae993672327_1.log'
```

Any idea where that's coming from?

**Comment by LivingInSyn** [comment]
> @LivingInSyn The snyk vanagon action [failed with](https://github.com/puppetlabs/puppet-runtime/runs/5211711881?check_suite_focus=true#step:4:1227)
> 
> ```
> Error: No space left on device : '/home/runner/runners/2.287.1/_diag/pages/67316a03-e358-4d3e-ac40-4186455a786e_04cbd90d-f885-551f-8636-bae993672327_1.log'
> ```
> 
> Any idea where that's coming from?

nope! That's a new one. Probably from the nginx caching, but it should be limited:

https://github.com/puppetlabs/security-snyk-vanagon-action/blob/main/local_nginx/nginx.conf#L1

File: .github/workflows/snyk_scan.yaml [hunk 1]
```diff
@@ -16,12 +13,19 @@ jobs:
         fetch-depth: 0
```
Comment (LivingInSyn) [inline .github/workflows/snyk_scan.yaml:13]: this `depth` should be `1`, not `0`, I messed up the original documentation

Comment (joshcooper) [reply .github/workflows/snyk_scan.yaml:13]: I may be misremembering, but I thought something in the `vanagon dependencies` command used a git ref like `HEAD` and that isn't available when using `fetch-depth: 1`. I'll switch it back to 0 to see what happens.

**Comment by joshcooper** [comment]
We're going to look at solving this a different way, closing for now.

**Pull request closed without merging**